src/Mutation/NodeTraverserFactory.php

```diff
@@ -49,20 +52,18 @@
 class NodeTraverserFactory
 {
     /**
-     * @param NodeVisitor[] $extraVisitors
+     * @param IgnoresNode[] $additionalIgnoredNodes
      */
-    public function create(array $extraVisitors): PrioritizedVisitorsNodeTraverser
+    public function create(NodeVisitor $mutationVisitor, array $additionalIgnoredNodes): NodeTraverser
     {
-        $traverser = new PrioritizedVisitorsNodeTraverser(new NodeTraverser());
-
-        $traverser->addPrioritizedVisitor(new NotMutableIgnoreVisitor(), 50);
-        $traverser->addPrioritizedVisitor(new ParentConnectorVisitor(), 40);
-        $traverser->addPrioritizedVisitor(new FullyQualifiedClassNameVisitor(), 30);
-        $traverser->addPrioritizedVisitor(new ReflectionVisitor(), 20);
-
-        foreach ($extraVisitors as $priority => $visitor) {
-            $traverser->addPrioritizedVisitor($visitor, $priority);
-        }
+        $traverser = new NodeTraverser();
+        $traverser->addVisitor(new NotMutableIgnoreVisitor(
+            array_merge($additionalIgnoredNodes, [new IgnoreInterfaceNode(), new IgnoreAbstractMethod()]))
```

Comment:
I don't think `array_merge()` is appropriate here: you could just append those two visitors directly to the array. That said, it's not a lot of additional nodes here anyway so I guess the performance impact does not really matter